test/tests/ghost-basics/run.sh

```diff
@@ -27,4 +27,10 @@ _request() {
 . "$dir/../../retry.sh" '_request GET / --output /dev/null'
 
 # Check that /ghost/ redirects to setup (the image is unconfigured by default)
-_request GET '/ghost/' -I | grep -q '^Location: .*setup'
+if [[ $serverImage == ghost:1* ]]
+then
+	_request GET '/ghost/api/v0.1/authentication/setup/' | grep -q 'status":false'
```

Comment:
I can't speak for the ghost dev team on this, but this API is pretty straightforward, it does one single thing and I saw no announcements about changes on this. But maybe @acburdine has some insight ? And for the version, it's true `v0.1` doesn't seam very stable but for now there is no `latest` tag :smile: 

Anyway, for the simple `GET` request, it wouldn't be enough as this URL still exists after setup it's the admin URL.